packages/arb-token-bridge-ui/src/components/MainContent/MainContent.tsx

```diff
@@ -40,28 +37,33 @@ export function MainContent() {
   }, [pendingTransactions, updatePendingTransaction])
 
   return (
-    <div className="flex w-full justify-center">
-      <div className="main-panel flex w-full max-w-[600px] flex-col gap-2">
-        <div className="hidden text-center text-5xl">Arbitrum Token Bridge</div>
+    <SidePanel
+      isOpen={isTransactionHistoryPanelVisible}
+      heading="Transaction History"
+      onClose={closeTransactionHistoryPanel}
+      scrollable={false}
+    >
+      <TransactionHistory props={{ ...transactionHistoryProps, address }} />
+    </SidePanel>
+  )
+}
 
+export function MainContent() {
+  return (
+    <>
+      <div className="main-panel mx-auto flex w-full max-w-[600px] flex-col sm:pb-12 sm:pt-6">
         <TransactionStatusInfo />
 
         <TransferPanel />
       </div>
-      <SidePanel
-        isOpen={isTransactionHistoryPanelVisible}
-        heading="Transaction History"
-        onClose={closeTransactionHistoryPanel}
-        scrollable={false}
-      >
-        <TransactionHistory props={{ ...transactionHistoryProps, address }} />
-      </SidePanel>
+
+      <TransactionHistorySidePanel />
 
       {/* Settings panel */}
       <SettingsDialog />
 
       {/* Toggle-able Stats for nerds */}
-      {isArbitrumStatsVisible && <ArbitrumStats />}
```

Comment:
moved condition into `<ArbitrumStats />`

Reply:
nit and general comment: stuff like this should stay outside of the component. the component itself should not need to worry about when or when it isn't rendered

Reply:
in principle, I agree for better readability
however, I do not agree from a performance perspective

react component rerenders when a state changes
so if we move `isArbitrumStatsVisible` into `ArbitrumStats`,
when `isArbitrumStatsVisible` changes, only `ArbitrumStats` is rerendered

but if we keep `isArbitrumStatsVisible` here in `MainContent`,
when `isArbitrumStatsVisible` changes, `MainContent` and `ArbitrumStats` will both rerender

Reply:
i'll move it back in on this PR
feels like there should be a separate conversation on performance and rerendering issues